mantidproject/mantid
# ISIS SANS v2 Made show transmission default

In the old sans GUI the fitted and unfitted transmission workspaces were output by default. This is not the default case in the new GUI currently but it has been requested that it is made the default behavior. This does that.

**To test:**
Get the loqdemo data from the ISIS Sample data here http://download.mantidproject.org/
Open the SANS ISIS V2 GUI switch the instrument to LOQ
Load MaskFile.txt as the user file and batch_mode_reduction.csv as the batch file.
On the left go to settings->adjustment and check that the show transmission box half way down the page is ticked. 
Go back to runs tab and click process.
Confirm that at the end of the reduction that four transmission workspaces are in the ADS one fitted and one unfitted for the sample and the can.


Fixes #22270 . 

**Release Notes** 
Release notes have been updated.


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
